source/addons-and-dependencies/managing-dependencies.md

```diff
@@ -33,12 +33,25 @@ without the need to `import` them.
 Provide the asset path as the first and only argument.
 
 ```ember-cli-build.js
-app.import('bower_components/moment/moment.js');
+app.import('node_modules/moment/moment.js');
 ```
 
 You will need to add `"moment"` to the `globals` section in `.eslintrc.js` to prevent ESLint errors
 about using an undefined variable.
 
+Alternatively, you can also use the amd transform with libraries that support both global and amd, to allow you to 
+`import` these assets.
+
+```ember-cli-build.js
+app.import('node_modules/moment/moment.js', {
+  using: [
+    { transformation: 'amd', as: 'moment' }
+  ]
+});
+```
+
+This tranformation allows you to `import` moment in your app. (e.g. `import moment from 'moment';`)
```

Comment:
CI failure is due to spellcheck - "transformation"

Reply:
Woah, nice! Had no idea we had spellcheck in CI. That's a great feature 😄